iOS-tic-tac-toeTests/GameViewControllerTests.swift

```diff
@@ -4,11 +4,20 @@ import Nimble
 
 class GameViewControllerSpec: QuickSpec {
     override func spec() {
-        let gameController = GameViewController()
-        
+        var gameController: GameViewController!
+        var mockBoardView: MockBoardView!
+        var mockStatusView: MockStatusView!
         let xMarker = UIImage(named: "letter-x")!
         let oMarker = UIImage(named: "letter-o")!
         let button = UIButton()
+        
+        beforeEach() {
+            mockBoardView = MockBoardView()
```

Comment:
I would expect more tests that make assertions against the `mockBoardView`. e.g. Making sure that the updated board is shown after a turn, making sure the all spot/moves are disabled when the game ends.